projects/proj-event-cards.md

```diff
@@ -0,0 +1,105 @@
+[← Back to Index](../index.md)
+
+# Frontend Mini Project - Event Cards
+
+## Week 1 - HTML
+
+Think of a really cool event: a concert, art show, festival, play, performance, meet-up, or anything else one may wish to attend. We want to get the word out about this awesome thing, so we are going to build some "digital flyers," or event cards.
+
+What sort of information would you want to know? How about:
+- Title of event or Band name
+- Picture of the band or location
+- Location of the event
+- When it is
+- Link to the website of the band, festival, or where you can buy tickets
+
+We want to build these cards with the following in mind:
+- A "Card" is a small piece of a website, not usually the whole page. We'll probably want to put multiple cards on the final page.
+- Put things in the order that makes the most sense for HTML.
+- Research semantic tags for each item; it is unlikely you'll be using `<div>` very often, but you may decide that it's the best choice some places.
+- Creating [accessible elements may require additional attributes](https://www.w3.org/WAI/standards-guidelines/wcag/).
+- Inline comments are helpful for reminding yourself why you made a decision you made.
+- Focus on semantically strong, accessible Markup. Styles will come later, and if you do them now, we'll probably completely ignore them anyway.
+
+Next week we will spend 5 minutes or so going over the HTML choices each of you have made. Try to do this individually (not as a group). Please do take advantage of office hours or pairing with developers!
```

Comment:
`office hours` is a thing we might want to link to in the sparkbox/standard.

Reply:
I  think we'll need to add this to the Standard, I'm not finding a section about office hours!